Implemented wand basic attack

JeromSar opened the pull request:
You can now use an emerald (wand) to shoot off basic attacks. The attack is passed to AttackManager whom may process the damage calculation.

Squash commit message:
```
Implement wand basic attack
Wands (currently emeralds) how have basic attacks. These have a sound and a particle effect. Particle effects are played by EffectLib, a library which has been added in source.
```

JeromSar committed:
Implemented wand basic attack
Added EffectLib to manage particle effects

JeromSar committed:
Add sound effects

JeromSar committed:
Limit wand use rate to 0.8 times per second

cashenkes reviewed (approved):
Not enough items but to many effects 👍

JeromSar committed:
Removed default EffectLib effects
We have custom effects for our wand basic attack, and these just bloat our final binary

JeromSar committed:
Merge branch 'devel' into feature/wand
